lib/bundle-helpers.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
```

Comment:
These are methods. So if you're going to break out some methods with a particular theme to a file in lib, go with `lib/bundle-methods.js`. Helpers implies something else (nunjucks helpers, which these are not).

Reply:
I think I just keep this function as a asset method as you suggested below. Makes much more sense.